samples/net/http_server/sample.yaml

```diff
@@ -12,14 +12,14 @@ tests:
     tags: net http
```

Comment:
Changes to http_server/sample.yaml do not seem to belong to patch called "net: llmnr_responder: Fix unused Warning", could you refactor this thing a bit?

Reply:
Refactored.